change-is-inevitable-versioning-eds/README.md

```diff
@@ -0,0 +1,14 @@
+# Change is inevitable: versioning event-driven systems
```

Comment:
An alternative:
```suggestion
# Dealing with the inevitable: versioning event-driven systems
```

Reply:
I think the first version connects better to the second abstract